src/lib/file-extractor.ts

```diff
@@ -200,4 +206,72 @@ function cleanupTempDir(dirPath: string) {
 export const replaceBase64Content = (data) => {
   // Remove all base64 encoded content from the "image" fields
     return data.replace(/data:image\/[a-zA-Z]+;base64,[a-zA-Z0-9+/=]+/g, "File content removed");
-};
\ No newline at end of file
+};
+
+
+
+export const processChatAttachments = async (messages: Message[]) => {
+  for (const message of messages) { // TODO: extract to a function so it can be re-used in the other chat endpoints
```

Comment:
@pkarw Is this `@TODO` satisfied since it's in `src/lib` and can be re-used in other endpoints, or did you mean further refactor the function in the iteration?

Reply:
Yes, you're right. Removing this comment